src/components/action-list/pages/action-list-interactive.jsx

```diff
@@ -3,7 +3,8 @@ import ActionList, {DIRECTION, ALIGNMENT} from '../ActionList';
 import ButtonSecondary, {BUTTON_SECONDARY_TYPE} from 'buttons/ButtonSecondary';
 import ButtonPrimary, {BUTTON_PRIMARY_TYPE} from 'buttons/ButtonPrimary';
 import Icon, {ICON_COLOR, TYPE as ICON_TYPE} from 'icons/Icon';
-import Text, {TYPE as TEXT_TYPE, COLOR as TEXT_COLOR} from 'text/Text';
+import Text, {TEXT_TYPE, TEXT_COLOR} from 'text';
```

Comment:
does it makes sense to have `index.js` in `components/text`? Wouldn't you rather have all the imports consistent like:
`text/Text`
`text/Link`
`text/Headline`

Reply:
@maciekbb maybe your right, I was hoping that maybe we would move components into separate folders eventually. But maybe this would be better to do, when I actually would move them.

Reply:
+1 to @maciekbb, I think if we want to restructure components into separate folders, we should do them all at once.